src/Components/Web.JS/src/Rendering/ElementReferenceCapture.ts

```diff
@@ -25,3 +35,14 @@ DotNet.attachReviver((key, value) => {
     return value;
   }
 });
+
+// Support return of the ElementRef from JS to .NET
+DotNet.attachReplacer((key, value) => {
+  if (value instanceof Element) {
+    const captureId = getCaptureIdFromElement(value);
+    if (captureId) {
+      return { [elementRefKey]: captureId };
+    }
+  }
+  return value;
+});
```

Comment:
This is adding some public API that I don't know we want (anything in `DotNet` is public API)

Reply:
Instead of doing it like this, could we "hijack" `JSObjectReference` to achieve this?

By that, I mean using the same key as
https://github.com/dotnet/aspnetcore/blob/c821481c641a08274e8bf9743eed10d1475d2c6b/src/JSInterop/Microsoft.JSInterop.JS/src/src/Microsoft.JSInterop.ts#L10-L14

And then on the C# side (if necessary) have a custom converter for `ElementReference`, or have it look for the `JSObjectIdKey` too if there's already one.